model-server/src/main/kotlin/org/modelix/model/server/Main.kt

```diff
@@ -192,6 +197,19 @@ object Main {
                     permissionSchema = ModelServerPermissionSchema.SCHEMA
                 }
                 install(ForwardedHeaders)
+                install(CallLogging) {
+                    format { call ->
+                        // Resemble the default format but include remote host and user agent for easier tracing on who issued a certain request.
```

Comment:
This fixes https://issues.modelix.org/issue/MODELIX-865/Enable-KTOR-request-logging-for-model-server

> Request should be logged at debug level.

By default `INFO` seems to be used.
See https://ktor.io/docs/server-call-logging.html#logging_level

> Also log thread ID for request, if possible

Can we add the thread ID?

Reply:
I wasn't aware of the ticket. Picked it up.

Why was it decided to deviate from the default log level in that ticket? Even if I disable all other logging, that's the most basic information I need to trace what's going on on the model-server. I wouldn't want to hide that in the debug-noise.

Reply:
Regarding the thread ID: I can log the thread ID at the time the call logging message is assembled. But that's not guaranteed to be the same thread that executed (parts of) the request. I don't think that helps but instead confuses more.

Reply:
I agree on both.
Lets just leave a note in the ticket why we deviated from the description.

Reply:
noted in the ticket.